### DISCUSSION TITLE
Function Plotting as AuxVariable

### DISCUSSION THREAD

**Discussion by makeclean** [idaholab/moose#17337] [Q&A General] [answered]
Is there an obvious way to plot the 3D results of a function? E.g. ive got a function which is applied as a body force, is there some way using AuxVariable that I can use to plot this data?

e.g.

```
  [heat]
    type = ADBodyForce
    variable = temp
    value = 1e6
    function = 'exp(-10*(y-0.13))'
    use_displaced_mesh = false
  []
```
Is there an obvious way to do this that I'm missing?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Use a [functionAux](https://mooseframework.inl.gov/source/auxkernels/FunctionAux.html) and plot the variable you store the function values in.
